Makefile

```diff
@@ -204,6 +204,15 @@ test-e2e-external-eks:
 	GINKGO_SKIP="\[Disruptive\]|\[Serial\]" \
 	./hack/e2e/run.sh
 
+.PHONY: test-helm-chart
+test-helm-chart:
+	AWS_REGION=us-west-2 \
+	AWS_AVAILABILITY_ZONES=us-west-2a,us-west-2b,us-west-2c \
+	EBS_INSTALL_SNAPSHOT="true" \
```

Comment:
Is this necessary? Does this test snapshots in any way?

Reply:
It's necessary. We run the `External.Storage` test suite with kubetest2 [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/662cbf898e74ad016f4b1d76cf5e9654131b70aa/charts/aws-ebs-csi-driver/templates/tests/helm-tester.yaml#L192) which includes the snapshot tests, if we don't install the snapshot controller and CRDs those tests will fail.

Technically, we could opt out of testing snapshot functionality here with `GINKGO_SKIP` but I couldn't think of a good reason for specifically skipping these tests.